docs/low-code-jobs/databricks-jobs.md

```diff
@@ -3,23 +3,165 @@ sidebar_position: 1
+job details (3).
+
+Whenever, creating a new job you're asked for the following details:
+
+| Field Name        | Description                                                                                                                                                                                                                                         |
+| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Project           | Which [project](/concepts/project) to create the Job in. This controls who has access to the Job, groups Jobs' together for lineage, and allows you to use Pipelines already published within that project.                                         |
+| Branch            | Which Git branch to use when developing this Job.                                                                                                                                                                                                   |
+| Name              | Unique job name.                                                                                                                                                                                                                                    |
+| Scheduler         | The underlying engine that's going to execute your job. Databricks is recommended.                                                                                                                                                                  |
+| Fabric            | The [execution fabric](/concepts/fabric) to which the job is going to be deployed.                                                                                                                                                                  |
+| Job Size          | The [default size](/concepts/fabric#whats-in-a-fabric) of the cluster, that's going to be created for the job to run.                                                                                                                               |
+| Schedule Interval | Defines how often your job is going to run. The interval is defined using the [Quartz format](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html). You can click on the clock icon, to easily pick the interval. |
+| Alerts email      | Comma separated list of emails, that are going to receive notifications on specific job status events (job start, failure, or success).                                                                                                             |
+
+### Building the job
+
+![Example Databricks Job](img/databricks-job-example.png)
+
+Now that you've created your first job, you can start adding the Gems onto the canvas to run them, during the job run
```

Comment:
Phrasing: "Now that you've created your first Job, you can start adding Gems to the canvas to define which Pipelines will be run during the Job. To define dependences between the Pipelines within the Job you can simply connect them by dragging and dropping a connection between them."